doc/en/components/grids/_shared/multi-column-headers.md

```diff
@@ -0,0 +1,280 @@
+---
+title: Multi-column Headers in {Platform} {ComponentTitle} - Infragistics
+_description: Start grouping column headers by placing them under a common hierarchical header with the help of Ignite UI for Angular grid and combine them into multi headers.
+sharedComponents: ["Grid", "TreeGrid", "HierarchicalGrid"]
+_keywords: []
```

Comment:
remove [] brackets, add a few keywords specific to the feature AND add {Platform}, {ComponentTitle}, {ComponentName}, {ProductName}, Infragistics